packages/core/core/test/Environment.test.js

```diff
@@ -6,7 +6,7 @@ import {createEnvironment} from '../src/Environment';
 describe('Environment', () => {
   it('assigns a default environment with nothing passed', () => {
     assert.deepEqual(createEnvironment(), {
-      id: '1051b306b4fe6e64',
+      id: '2163092a3dd83a5f',
```

Comment:
😞

Reply:
Would be neat if we used/made something along the lines of Jest's stringMatching matcher: https://jestjs.io/docs/expect#expectstringmatchingstring--regexp